src/taipy/gui/viselements.json

```diff
@@ -758,8 +758,9 @@
           "properties": [
             {
                 "name": "lov",
+                "default_property": true,
                 "type": "dict[str, any]",
```

Comment:
Why isn't this `dict[str, str]`, reading the doc?

Reply:
Ok so the doc is, say incomplete...

Reply:
same as lovComp.lov